client/src/components/pages/MembersPage/components/CreateForm/CreateForm.js

```diff
@@ -0,0 +1,186 @@
+import React, { useRef } from 'react'
+import {
+  Form,
+  Row,
+  RowFlexStart,
+  CheckboxRow,
+  CheckboxDescription,
+  Column,
+  FormTitle,
+  InputField,
+  BandSelector,
+  BandSelectOption,
+  Tag,
+  AddNewMembersInstrument,
+  CloseForm,
+  Submit,
+} from './CreateFormStyledComponents.js'
+
+function CreateForm({
+  members,
+  setMembers,
+  postMember,
+  newMembersInstruments,
+  addNewMembersInstrument,
+  removeNewMembersInstrument,
+  resetNewMembersInstruments,
+  bands,
+  setBands,
+  toggleForm,
+}) {
+  const newInstrumentInputRef = useRef()
+
+  return (
+    <Form onSubmit={createMember}>
+      <CloseForm onClick={toggleForm}>
+        <i className="fas fa-times-circle" style={{ background: '0' }}></i>
+      </CloseForm>
+      <FormTitle>
+        <h3>Create Form</h3>
+      </FormTitle>
+
+      <InputField
+        type="text"
+        name="memberName"
+        placeholder="New member's name"
+        autoFocus
+      />
+      <Column>
+        <BandSelector name="band">
+          {bands.map(band => (
+            <BandSelectOption value={band.value} key={band.value}>
+              {band.name}
+            </BandSelectOption>
+          ))}
+        </BandSelector>
+
+        <InputField type="text" name="newBand" placeholder="new band" />
+      </Column>
+      <Row>
+        <Column>
+          <CheckboxDescription>Choose new member's roles:</CheckboxDescription>
+          <Row>
+            <CheckboxRow>
+              <label htmlFor="administrator">
+                <input
+                  type="checkbox"
+                  id="administrator"
+                  name="administrator"
+                  value="administrator"
+                />
+                Administrator
+              </label>
+            </CheckboxRow>
+            <CheckboxRow>
+              <label htmlFor="musician">
+                <input
+                  type="checkbox"
+                  id="musician"
+                  name="musician"
+                  value="musician"
+                />
+                Musician
+              </label>
+            </CheckboxRow>
+          </Row>
+        </Column>
+      </Row>
+      <Row>
+        <InputField
+          type="text"
+          name="newInstrument"
+          placeholder="new instrument"
+          ref={newInstrumentInputRef}
+        />
+        <AddNewMembersInstrument
+          onClick={event => {
+            event.preventDefault()
+            newInstrumentInputRef.current.value &&
+              addNewMembersInstrument(newInstrumentInputRef.current.value)
+            newInstrumentInputRef.current.value = ''
+            newInstrumentInputRef.current.focus()
+          }}
+        >
+          <i className="fas fa-plus-circle" style={{ background: '0' }}></i>
+        </AddNewMembersInstrument>
+      </Row>
+      <RowFlexStart>
+        {newMembersInstruments.map(instrument => (
+          <Tag
+            key={instrument}
```

Comment:
`instrument` is supplied by user input. Is there some logic to prevent entering the same instrument twice? If not, this is a bad value to be used as a key prop.